pipeline/runtime/common.go

```diff
@@ -117,6 +117,9 @@ func fetchExportedVarsFromEnvFile(envFile string, out io.Writer, useCINewGodotEn
 
 	if useCINewGodotEnvVersion {
 		env, err = v3.Read(envFile)
+		if err != nil {
```

Comment:
Add a comment above mentioning fallback to old godotenv version

Reply:
@raghavharness sorry, I missed your comment